### PR TITLE
[#31] Fix error message printing

### PR DESCRIPTION
**Pull request by aschenbecherwespe**
Also remove an unneeded default asignment and unneeded linter disable

### CODE REVIEW

**Review by flipback — CHANGES_REQUESTED** [review]
Looks great! Only two things:
1. CHANGELOG.md
2.  I think we can add a test.

File: pkg/reduct/error.py [hunk 1]
```diff
@@ -14,8 +14,8 @@ class ReductError(Exception):
     def __init__(self, code: int, message: str):
         self._code = code
         self._detail = message
```
Comment (flipback) [inline pkg/reduct/error.py:16]: Looks like unused

**Review by flipback — CHANGES_REQUESTED** [review]

**Comment by aschenbecherwespe** [comment]
Create a new test?

File: pkg/reduct/error.py (hunk 1 above)
Comment (aschenbecherwespe) [reply pkg/reduct/error.py:16]: Yeah, I'm not sure. Are any of these used internally by python?

Comment (flipback) [reply pkg/reduct/error.py]: I think not.

Comment (aschenbecherwespe) [reply pkg/reduct/error.py]: okay removed

**Review by flipback — APPROVED** [review]

**Pull request merged**